apps/console/src/features/core/components/advanced-search-with-basic-filters.tsx

```diff
@@ -241,6 +241,8 @@ export const AdvancedSearchWithBasicFilters: FunctionComponent<AdvancedSearchWit
             submitted={ isFormSubmitted }
             dropdownPosition={ dropdownPosition }
             triggerClearQuery={ triggerClearQuery }
+            defaultFilterConditionOptions = { defaultFilterConditionOptions }
```

Comment:
The attribute should be `filterConditionOptions` here. If `filterConditionOptions` is not overriden from outside, we should use the `defaultFilterConditionOptions`. 

```suggestion
            filterConditionOptions = { filterConditionOptions || defaultFilterConditionOptions }
```